modules/bazel-diff/7.2.13/MODULE.bazel

```diff
@@ -0,0 +1,32 @@
+module(
+    name = "bazel-diff",
+    version = "7.2.13",
+    compatibility_level = 0,
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.6.1")
+bazel_dep(name = "rules_proto", version = "6.0.0-rc2")
+bazel_dep(name = "rules_java", version = "7.6.5")
+bazel_dep(name = "rules_kotlin", version = "1.9.1")
+bazel_dep(name = "rules_jvm_external", version = "6.0")
+
+bazel_diff_maven = use_extension("@rules_jvm_external//:extensions.bzl", "maven")
+bazel_diff_maven.install(
+    artifacts = [
+        "com.google.code.gson:gson:jar:2.9.0",
+        "com.google.guava:guava:31.1-jre",
+        "com.willowtreeapps.assertk:assertk-jvm:0.25",
+        "info.picocli:picocli:jar:4.6.3",
+        "io.insert-koin:koin-core-jvm:3.1.6",
+        "io.insert-koin:koin-test-junit4:4.0.0",
+        "junit:junit:5.11.2",
+        "org.apache.commons:commons-pool2:2.11.1",
+        "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.5.2",
+        "org.mockito.kotlin:mockito-kotlin:5.4.0",
+    ],
+    lock_file = "//:maven_install.json",
+)
+use_repo(
+    bazel_diff_maven,
+    bazel_diff_maven = "maven",
```

Comment:
BTW, you can write `use_repo(maven, maven = "bazel_diff_maven")` here to avoid having to change all your references to `@maven` in your BUILD files. But it looks like you already changed those, so this is up to you.